packages/internals/src/__tests__/getGenerators/__snapshots__/getGenerators.test.ts.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`getGenerators fail if generator not found 1`] = `"The generator [1minvalid_generator[22m specified via [1m--generator[22m does not exist in your Prisma schema"`;
```

Comment:
```suggestion
exports[`getGenerators fail if generator not found 1`] = `"The generator invalid_generator specified via --generator does not exist in your Prisma schema"`;
```

Reply:
Because test fails with https://github.com/prisma/prisma/actions/runs/3742671182/jobs/6353907985#step:10:1002
```
    Snapshot: "The generator invalid_generator specified via --generator does not exist in your Prisma schema"
    Received: "The generator invalid_generator specified via --generator does not exist in your Prisma schema"
```